src/mono/sample/wasm/browser-bench/runtime.js

```diff
@@ -1,23 +1,49 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
-
 var Module = { 
+    config: null,
```

Comment:
Why do we have multiple copies of the runtime.js ?

Reply:
Are all of them same ?

Reply:
If they are not same and we expect user to own such code for each project, could we re-factor repeated code ?

Reply:
The runtime.js file is the main entry point for the js in the user code. So there is one per sample. Since I modified the flow through the user code, I had to modify all of them. Hence, you see all of them in this PR. 

I don't think they can be removed as each sample is supposed to be independent I think. 

Do you have any ideas for re-factorings? I'm open for suggestions :).

Reply:
so, maybe the file should be names `main.js` and really be something that user owns. In such case any methods which are called on it as part of our API.

Reply:
I agree `main.js` is probably clearer. Where can I find our API docs? You are right I should probably add this `onConfigLoaded` callback into them. It is new, all the other methods are unchanged by this PR.

Reply:
That API is yet undocumented. We could create `main.d.ts` file for it all all other things we expect.

If this is main.js shall we let the user to name `onConfigLoaded` to whatever he wants and only document `Module.config`  and perhaps `js_support` `load_config()` ?

Reply:
But even that's not ideal I think. 'Module' is very generic name for something we would like to claim in global namespace.

Reply:
Unfortunately renaming onConfigLoaded would be quite difficult I think as it is called as the first thing in dotnet.js via `Module['onConfigLoaded'](config)`. So I would need to somehow guarantee that I get some parameter before the callback happens. Perhaps that could be done by adding a new field into mono-config.json that the user can modify. Is this worth it though? I think it might make it a bit more confusing actually and maybe make the mono-config,json file even less elegant. What if we just pick a better name than `onConfigLoaded`?

@lewing what do you think about this idea?

Reply:
I have another PR to rethink this behavior as it is a bit out of scope of this PR: #https://github.com/Daniel-Genkin/runtime/pull/2